gui/src/store/settings.ts

```diff
@@ -5,51 +5,39 @@ import { subscribeWithSelector } from "zustand/middleware";
 
 import { GeneralSettings } from "@iron/types/settings";
 
-interface State {
+interface Store {
   settings?: GeneralSettings;
   actions: Action[];
-}
 
-interface Setters {
   reload: () => void;
-  reloadActions: () => void;
 }
 
-type Store = State & Setters;
+const actionId = "settings/fastMode";
 
-const store: StateCreator<Store> = (set, get) => ({
+const store: StateCreator<Store> = (set) => ({
   settings: undefined,
-  actions: [],
+  actions: [
+    {
+      id: actionId,
+      name: "Fast mode",
+      subtitle: "enable/disable",
+      shortcut: ["˄ ", "+", " F"],
+    },
+    ...(["Enable", "Disable"] as const).map((mode, index) => ({
+      id: `${actionId}/${mode}`,
+      name: `${index + 1}: ${mode}`,
+      parent: actionId,
+      perform: () => {
+        const currentMode = mode === "Disable" ? false : true;
+        invoke("settings_set_fast_mode", { mode: currentMode });
```

Comment:
```suggestion
        invoke("settings_set_fast_mode", { mode: mode == "Enable" });
```

You don't really a ternary expression here since `a === b` already returns a boolean 😄